GoogleUtilities/Example/Tests/Swizzler/GULSceneDelegateSwizzlerTest.m

```diff
@@ -0,0 +1,140 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <GoogleUtilities/GULSceneDelegateSwizzler.h>
+#import "GULSceneDelegateSwizzler_Private.h"
+
+#import <OCMock/OCMock.h>
+#import <XCTest/XCTest.h>
+#import <objc/runtime.h>
+
+#pragma mark - Scene Delegate
+
+#if UISCENE_SUPPORTED
+
+@protocol TestSceneProtocol <UISceneDelegate>
+@end
+
+API_AVAILABLE(ios(13.0), tvos(13.0))
+@interface GULTestSceneDelegate : NSObject <UISceneDelegate>
+@end
+
+@implementation GULTestSceneDelegate
+@end
+
+@interface GULSceneDelegateSwizzlerTest : XCTestCase
+@end
+
+@implementation GULSceneDelegateSwizzlerTest
+
+- (void)testProxySceneDelegateWithNoSceneDelegate {
+  if (@available(iOS 13, tvOS 13, *)) {
```

Comment:
can we only target some tests on some particular iOS > 12? This seems to be lots of ugly checks for each test. If not, please have an early return check 
```
if (!@available(iOS 13, tvOS 13, *) { return ;}
...
```

Reply:
I just tried early return, however the complier raises warnings since the ios 13+ apis are not guarded with `@available`.

`@available does not guard availability here; use if (@available) instead`